pkg/volume/awsebs/attacher.go

```diff
@@ -43,13 +43,13 @@ type awsElasticBlockStoreAttacher struct {
 	awsVolumes aws.Volumes
 }
 
-var _ volume.Attacher = &awsElasticBlockStoreAttacher{}
+var _ volume.Attacher = (*awsElasticBlockStoreAttacher)(nil)
```

Comment:
We have this code all over Kubernetes. I'd prefer to have it consistent, so either change it everywhere in this repo or keep it as it is.

The other changes looks OK.

Reply:
Thanks for the reply, I will remove the interface check changes